app/controllers/manual_document_attachments_controller.rb

```diff
@@ -1,6 +1,17 @@
+require "create_manual_document_attachment_service"
+require "update_manual_document_attachment_service"
+require "show_manual_document_attachment_service"
+require "new_manual_document_attachment_service"
+
 class ManualDocumentAttachmentsController < ApplicationController
   def new
-    manual, document, attachment = services.new(self).call
+    service = NewManualDocumentAttachmentService.new(
+      repository,
+      # TODO: This be should be created from the document or just be a form object
```

Comment:
Yeah. I think this should be dealt with separately - I think it's clearer if this code matches the code which it was taken from as closely as possible, i.e. the typo in the original is exactly replicated here.